src/Schema/Validation/ControllerPathValidation.php

```diff
@@ -22,12 +22,9 @@ protected function validateSlashes(SchemaBuilder $builder): void
 		foreach ($controllers as $controller) {
 			$path = $controller->getPath();
 
-			if (strlen($path) === 1) {
-				if ($path === '/') continue;
-
-				// MUST: Be exactly /
+			if ($path === null) {
```

Comment:
Why have you changed it?

Reply:
Exception was misleading. @ControllerPath does not must be `/`, it just could be only `/`. Check if path starts with slash is enough for that.

Reply:
Ouk